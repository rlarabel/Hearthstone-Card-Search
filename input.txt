BEEEES!!!
Flamestrike
Notareal Card
Slep Bunwalla
Goliath, Sneed's Masterpiece
Leeroy Jenkins
q

