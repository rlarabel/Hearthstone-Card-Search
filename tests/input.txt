The Demon Seed
Bloodscalp Strategist
Blood Imp
Mindgames
Ice Barrier
Self-Sharpening Sword
Sathrovarr
Southsea Deckhand
Equality
Jan'alai, the Dragonhawk
Noble Mount
Fleethoof Pearltusk
Also Missing
Eldritch Horror
Nerubian Egg
Deathwing, Mad Aspect
Something Wrong
Ancient of War
Cursed Vagrant
Grimscale Chum
q

