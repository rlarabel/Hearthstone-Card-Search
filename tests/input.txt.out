>> The Demon Seed
The Demon Seed                 1
Spell       Warlock    Legendary
--------------------------------
[1mQuestline:[0m Take 8
damage on your turns.
[1mReward:[0m [1mLifesteal[0m. Deal $3
damage to the enemy hero.
--------------------------------
0                              0

>> Bloodscalp Strategist
Bloodscalp Strategist          3
Minion       Hunter         Rare
--------------------------------
[1mBattlecry:[0m If you have a weapon equipped, [1mDiscover[0m a spell.
--------------------------------
2                              4

>> Blood Imp
Blood Imp                      1
Minion      Warlock       Common
--------------------------------
  [1mStealth[0m. At the end of your  
turn, give another random
 friendly minion +1 Health.
--------------------------------
0                              1

>> Mindgames
Mindgames                      4
Spell        Priest         Epic
--------------------------------
Put a copy of
a random minion from
your opponent's deck into the battlefield.
--------------------------------
0                              0

>> Ice Barrier
Ice Barrier                    3
Spell         Mage        Common
--------------------------------
[1mSecret:[0m When your
hero is attacked,
gain 8 Armor.
--------------------------------
0                              0

>> Self-Sharpening Sword
Self-Sharpening Sword          3
Weapon       Rogue          Rare
--------------------------------
After your hero attacks, gain +1 Attack.
--------------------------------
1                              0

>> Sathrovarr
Sathrovarr                     9
Minion      Neutral    Legendary
--------------------------------
[1mBattlecry:[0m Choose a friendly minion. Add a copy of it to your hand, deck, and battlefield.
--------------------------------
5                              5

>> Southsea Deckhand
Southsea Deckhand              1
Minion      Neutral       Common
--------------------------------
Has [1mCharge[0m while you have a weapon equipped.
--------------------------------
2                              1

>> Equality
Equality                       3
Spell       Paladin         Rare
--------------------------------
Change the Health of ALL minions to 1.
--------------------------------
0                              0

>> Jan'alai, the Dragonhawk
Jan'alai, the Dragonhawk       7
Minion        Mage     Legendary
--------------------------------
[1mBattlecry:[0m If your Hero Power
dealt 8 damage this game,
summon Ragnaros the
Firelord.
--------------------------------
4                              4

>> Noble Mount
Noble Mount                    2
Spell       Paladin         Rare
--------------------------------
Give a minion +1/+1
and [1mDivine Shield[0m.
When it dies, summon
a Warhorse.
--------------------------------
0                              0

>> Fleethoof Pearltusk
Fleethoof Pearltusk            5
Minion      Neutral       Common
--------------------------------
[1mRush[0m
[1mCorrupt:[0m Gain +4/+4.
--------------------------------
4                              4

>> Also Missing
./search: 'Also Missing' not found!
>> Eldritch Horror
Eldritch Horror                8
Minion      Neutral       Common
--------------------------------

--------------------------------
6                             10

>> Nerubian Egg
Nerubian Egg                   2
Minion      Neutral         Rare
--------------------------------
[1mDeathrattle:[0m Summon a 4/4 Nerubian.
--------------------------------
0                              2

>> Deathwing, Mad Aspect
Deathwing, Mad Aspect          8
Minion      Warrior    Legendary
--------------------------------
[1mBattlecry:[0m Attack ALL
other minions.
--------------------------------
12                            12

>> Something Wrong
./search: 'Something Wrong' not found!
>> Ancient of War
Ancient of War                 7
Minion       Druid          Epic
--------------------------------
[1mChoose One -[0m
+5 Attack; or +5 Health and [1mTaunt[0m.
--------------------------------
5                              5

>> Cursed Vagrant
Cursed Vagrant                 7
Minion       Rogue        Common
--------------------------------
[1mDeathrattle:[0m Summon a 7/5 Shadow with [1mStealth[0m.
--------------------------------
7                              5

>> Grimscale Chum
Grimscale Chum                 1
Minion      Paladin       Common
--------------------------------
[1mBattlecry:[0m Give a random
Murloc in your hand +1/+1.
--------------------------------
2                              1

>> q
